apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Snippets/PublisherClientSnippets.cs

```diff
@@ -49,7 +45,8 @@ public void AddCustomizedPublisherClient()
         services.AddPublisherClient(builder =>
         {
             builder.TopicName = topicName;
-            builder.CredentialsPath = "path/to/credentials.json";
+            // An instance of Google.Apis.Auth.OAuth2.GoogleCredential that you can create using Google.Apis.Auth.OAuth2.CredentialFactory and other GoogleCredential factory methods.
```

Comment:
The access token credential is not that usual or useful, so I rather not show that part in the snippet, which starts right after the `// Sample: ...` comment.

Initialize the GoogleCredential outside of the snippet, in the same place that the projectId, etc. is being initialized, and assign it to a googleCredential variable. Then use that variable her and keep the comment as is. Sorry if I was not clear earlier.